tests/testthat/test_base_measures.R

```diff
@@ -786,6 +786,16 @@ test_that("getDefaultMeasure", {
   expect_equal(mmce, getDefaultMeasure("classif"))
 })
 
+test_that("measure properties", {
+  #hasMeasureProps yields correct properties
+  expect_true(all(vlapply(listMeasures(create = TRUE),
+    function(m) all(hasMeasureProperties(m, m$properties)))))
+  props = listMeasureProperties()
+  #all props exist in mlr$measure.properties
+  expect_true(all(vlapply(listMeasures(create = TRUE),
+    function(m) all(getMeasureProperties(m) %in% props))))
```

Comment:
Could you check here whether the returned list of properties is not empty please? Same above for the other test.